.travis.yml

```diff
@@ -17,6 +17,7 @@ deploy:
   script: bash scripts/deploy.sh
   on:
     tags: true
-
-
-
+branches:
+  only:
+  - master
+  - /\d{1,3}\.\d{1,3}\.\d{1,3}/
```

Comment:
Do we have this in our other repos? Should we?

Reply:
Yes to the latter, no to the former.

Reply:
I think we were going to take care of a few and scout the rest